migtests/scripts/config-templates/live-migration-with-fall-back.yaml

```diff
@@ -0,0 +1,88 @@
+export-dir: {{ EXPORT_DIR }}
+
+source:
+  db-type: {{ SOURCE_DB_TYPE }}
+  {% if SOURCE_DB_ORACLE_TNS_ALIAS is not defined or SOURCE_DB_ORACLE_TNS_ALIAS == "" %}
+  db-host: {{ SOURCE_DB_HOST }}
+  db-port: {{ SOURCE_DB_PORT }}
+  {% endif %}
+  db-name: {{ SOURCE_DB_NAME }}
+  {% if SOURCE_DB_TYPE != "mysql" %}
+  db-schema: {{ SOURCE_DB_SCHEMA }}
+  {% endif %}
+  db-user: {{ SOURCE_DB_USER }}
+  db-password: {{ SOURCE_DB_PASSWORD }}
+  
+  {% if SOURCE_DB_SSL_MODE is defined and SOURCE_DB_SSL_MODE != "" %}
+  ssl-mode: {{ SOURCE_DB_SSL_MODE }}
+  {% endif %}
+
+  {% if SOURCE_DB_SSL_CERT is defined and SOURCE_DB_SSL_CERT != "" %}
+  ssl-cert: {{ SOURCE_DB_SSL_CERT }}
+  {% endif %}
+
+  {% if SOURCE_DB_SSL_KEY is defined and SOURCE_DB_SSL_KEY != "" %}
+  ssl-key: {{ SOURCE_DB_SSL_KEY }}
+  {% endif %}
+
+  {% if SOURCE_DB_SSL_ROOT_CERT is defined and SOURCE_DB_SSL_ROOT_CERT != "" %}
+  ssl-root-cert: {{ SOURCE_DB_SSL_ROOT_CERT }}
+  {% endif %}
+
+  {% if SOURCE_DB_ORACLE_TNS_ALIAS is defined and SOURCE_DB_ORACLE_TNS_ALIAS != "" %}
+  oracle-tns-alias: {{ SOURCE_DB_ORACLE_TNS_ALIAS }}
+  {% endif %}
+  {% if SOURCE_DB_ORACLE_CDB_TNS_ALIAS is defined and SOURCE_DB_ORACLE_CDB_TNS_ALIAS != "" %}
+  oracle-cdb-tns-alias: {{ SOURCE_DB_ORACLE_CDB_TNS_ALIAS }}
+  {% endif %}
+  {% if SOURCE_DB_TYPE == "oracle" %}
+  oracle-cdb-name: {{ ORACLE_CDB_NAME }}
+  {% endif %}
+
+target:
+  db-host: {{ TARGET_DB_HOST }}
+  db-port: {{ TARGET_DB_PORT }}
+  db-name: {{ TARGET_DB_NAME }}
+  {% if SOURCE_DB_TYPE != "postgresql" %}
+  db-schema: {{ TARGET_DB_SCHEMA }}
+  {% endif %}
+  db-user: {{ TARGET_DB_USER }}
+  db-password: {{ TARGET_DB_PASSWORD }}
+
+assess-migration:
+  iops-capture-interval: 0
+
+export-data:
```

Comment:
Nit: Should we make it export-data-from-source in the live templates. It doesn't change the functionality but to be consistent with the suggested workflow in the docs?